### DISCUSSION TITLE
How to style the options layout of a select element?

### DISCUSSION THREAD

**Discussion by alamenai** [tailwindlabs/tailwindcss#10625] [Help]
I want to style the `options` layout of the `select` element, however, I don't know what classes I should apply to change its style.

![Screenshot from 2023-02-18 16-18-49](https://user-images.githubusercontent.com/20143684/219873860-2fc8af90-1f06-4b56-b08c-f44e0405473a.png)

**Comment by wongjn** [comment]
There's not a lot you can do to style `<option>` elements within a `<select>` element, [from MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#styling_with_css):

> Styling the `<option>` element is highly limited. Options don't inherit the font set on the parent. In Firefox, only [`color`](https://developer.mozilla.org/en-US/docs/Web/CSS/color) and [`background-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color) can be set however in Chrome or Safari it's not possible to set any properties. You can find more details about styling in [our guide to advanced form styling](https://developer.mozilla.org/en-US/docs/Learn/Forms/Advanced_form_styling).
